_data/nav.yml

```diff
@@ -55,3 +63,31 @@ home:
   url: /releases/
 - label: Sitemap
   url: /sitemap/
+
+install:
```

Comment:
THese should come from the new install page, right? Or is this different?

Reply:
No, that's different. This navigation is between the sub-pages. The install page itself additionally has a TOC.